src/executor.rs

```diff
@@ -647,15 +647,24 @@ fn parse_result(
             Some(return_ptr) => JitValue::from_jit(return_ptr, type_id, registry),
             None => {
                 #[cfg(target_arch = "x86_64")]
-                let value = JitValue::from_jit(return_ptr.unwrap(), type_id, registry);
+                unreachable!();
```

Comment:
Can you please add a comment explaining the difference behaviour across architectures?

Reply:
Something like this should work:
```suggestion
                // Since x86_64's return values hold at most two different 64bit registers,
                // everything bigger than u128 will be returned by memory, therefore making
                // this branch unreachable on that architecture.
                unreachable!();
```

Reply:
Great!
Done [9d9b7a8](https://github.com/lambdaclass/cairo_native/pull/610/commits/9d9b7a8c8da573d23f8aec9a35f75191bebece52)